crates/libm-test/src/lib.rs

```diff
@@ -0,0 +1,65 @@
+extern crate core;
```

Comment:
I think you might be able to stamp a top-level `!#[cfg(test)]` here for the whole crate.